packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -121,26 +138,18 @@ const CheckoutPage = props => {
                             : 'Review and Place Order'}
```

Comment:
I understand that this expression was already here, not introduced by this PR, but when you see a multiline expression in a component you're working in, might as well fix it. 🙂

Reply:
Didn't see it, can fix though.